src/go/intelligence/models/models.go

```diff
@@ -121,11 +121,17 @@ type silence struct {
 	SilenceStatus []string `json:"silenceStatus"` //Labels for status of the silence
 }
 
+//annotations struct for storing specific set of keywords & set of dashboards to annotate when these keywords are matched in alerts.
+type annotationsData struct {
+	Actions []string `json:"actions"`
```

Comment:
please add documentation strings for EACH member of the struct

Reply:
added